src/styles/details_list.scss

```diff
@@ -3,6 +3,16 @@
   flex-direction: column;
 }
 
+.Layer__details-list__actions > .Layer__details-list__close-btn {
+  width: 36px;
+  height: 36px;
+  padding: 0;
+
+  @container (max-width: 1025px) {
```

Comment:
most small viewports have 1024 should we use 1025?

Reply:
Fixed, thanks for catching this!